x-pack/plugins/observability/public/components/shared/exploratory_view/configurations/lens_attributes.ts

```diff
@@ -501,14 +501,15 @@ export class LensAttributes {
 
   getMainYAxis(layerConfig: LayerConfig, layerId: string, columnFilter: string) {
     const { breakdown } = layerConfig;
-    const { sourceField, operationType, label } = layerConfig.seriesConfig.yAxisColumns[0];
+    const { sourceField, operationType, label, timeScale } =
+      layerConfig.seriesConfig.yAxisColumns[0];
 
     if (sourceField === RECORDS_PERCENTAGE_FIELD) {
       return getDistributionInPercentageColumn({ label, layerId, columnFilter }).main;
     }
 
     if (sourceField === RECORDS_FIELD || !sourceField) {
-      return this.getRecordsColumn(label);
+      return this.getRecordsColumn(label, undefined, timeScale);
```

Comment:
Just a note to say this `undefined` here probably shouldn't be `undefined`. The `getRecordsColumn` function takes a `columnFilter` argument here, but it looks to me like there was once one overall function for handling getting the columns, but then at some point some control flow statements were added which meant certain source fields were handled differently, and somewhere along the line the `columnFilter` and `timeScale` were lost. I tried to add this filter back but it wasn't 100% obvious how it should work. So this just adds back the `timeScale`. This probably warrants an issue filed for the Exploratory View.